.templatesyncignore

```diff
@@ -1,15 +1,17 @@
 # THIS FILE CANNOT BE SYNCED
 # use glob patterns as in .gitignore
 # recommend listing non-basics tutorial directories here
-
+###
 # configuration files
 cookiecutter.yaml
-
+###
 # environment lock files
-./conda/*lock.yml
-
+conda/*lock.yml
+###
+# team member files
+team/*-*
```

Comment:
Maybe keep the `header.yaml`?
I am not sure this is critical though.

Also interesting syntax `*-*`. Looks like an old school emoji :laughing:

Reply:
> Maybe keep the `header.yaml`?

As in, don't update this from the template?

> Also interesting syntax *-*. Looks like an old school emoji 😆

Haha - yup. But it made it so `header.yaml`, `README.md`, and `template.yaml` could be updated without my listing all of our yamls out...